--- matplotlib-0.90.1/setup.py.tkagg	2007-06-04 10:34:46.000000000 -0600
+++ matplotlib-0.90.1/setup.py	2007-06-04 10:54:44.000000000 -0600
@@ -28,13 +28,13 @@
 # it.  It makes very nice antialiased output and also supports alpha
 # blending
 BUILD_AGG          = 1
-BUILD_GTKAGG       = 'auto'
-BUILD_GTK          = 'auto'
+BUILD_GTKAGG       = 1
+BUILD_GTK          = 1
 
 # build TK GUI with Agg renderer ; requires Tkinter Python extension
 # and Tk includes
 # Use False or 0 if you don't want to build
-BUILD_TKAGG        = 'auto'
+BUILD_TKAGG        = 1
 
 # build wxPython extension code to efficiently blit agg into wx.  Only
 # needed for wxpython <2.8 if you plan on doing animations
@@ -226,11 +226,11 @@
 
 havegtk.gotit = None
 
-if BUILD_GTK and havegtk():
+if BUILD_GTK:
     build_gdk(ext_modules, packages, NUMERIX)
     rc['backend'] = 'GTK'
 
-if BUILD_GTKAGG and havegtk():
+if BUILD_GTKAGG:
     BUILD_AGG = 1
     build_gtkagg(ext_modules, packages, NUMERIX)
     rc['backend'] = 'GTKAgg'
@@ -245,16 +245,9 @@
         print 'Tkinter present but import failed'
         BUILD_TKAGG = 0
     else:
-        try:
-            tk = Tkinter.Tk()
-            tk.withdraw()
-        except Tkinter.TclError:
-            print 'Tkinter present, but window failed to open'
-            BUILD_TKAGG = 0
-        else:
-            BUILD_AGG = 1
-            build_tkagg(ext_modules, packages, NUMERIX)
-            rc['backend'] = 'TkAgg'
+        BUILD_AGG = 1
+        build_tkagg(ext_modules, packages, NUMERIX)
+        rc['backend'] = 'TkAgg'
 
 if BUILD_WXAGG:
     try:
