--- matplotlib-0.87.7/setup.py.pygtk	2006-12-05 11:13:58.000000000 -0700
+++ matplotlib-0.87.7/setup.py	2006-12-05 11:31:22.000000000 -0700
@@ -207,8 +207,9 @@
         print 'GTK requires pygtk'
         BUILD_GTK = 0
     except RuntimeError:
-        print 'pygtk present but import failed'
-        BUILD_GTK = 0
+        BUILD_GTK = 1
+        build_gdk(ext_modules, packages, NUMERIX)
+        rc['backend'] = 'GTK'
     else:
         version = (2,2,0)
         if gtk.pygtk_version < version:
@@ -228,7 +229,9 @@
         BUILD_GTKAGG=0
     except RuntimeError:
         print 'pygtk present but import failed'
-        BUILD_GTKAGG = 0
+        BUILD_GTKAGG = 1
+        build_gtkagg(ext_modules, packages, NUMERIX)
+        rc['backend'] = 'GTKAgg'
     else:
         version = (2,2,0)
         if gtk.pygtk_version < version:
