diff -up matplotlib-1.2.0/lib/matplotlib/font_manager.py.orig matplotlib-1.2.0/lib/matplotlib/font_manager.py
--- matplotlib-1.2.0/lib/matplotlib/font_manager.py.orig	2013-01-16 16:10:23.104026087 -0200
+++ matplotlib-1.2.0/lib/matplotlib/font_manager.py	2013-01-16 18:44:12.691379520 -0200
@@ -62,7 +62,7 @@ try:
 except ImportError:
     import pickle
 
-USE_FONTCONFIG = False
+USE_FONTCONFIG = True
 
 verbose = matplotlib.verbose
 
@@ -771,7 +771,7 @@ class FontProperties(object):
                 return float(self._size)
             except ValueError:
                 pass
-        default_size = fontManager.get_default_size()
+        default_size = FontManager.get_default_size()
         return default_size * font_scalings.get(self._size)
 
     def get_file(self):
@@ -991,7 +991,10 @@ class FontManager:
         self.afmfiles = findSystemFonts(paths, fontext='afm') + \
             findSystemFonts(fontext='afm')
         self.afmlist = createFontList(self.afmfiles, fontext='afm')
-        self.defaultFont['afm'] = self.afmfiles[0]
+        try:
+            self.defaultFont['afm'] = self.afmfiles[0]
+        except IndexError:
+            self.defaultFont['afm'] = None
 
         self.ttf_lookup_cache = {}
         self.afm_lookup_cache = {}
@@ -1002,7 +1005,8 @@ class FontManager:
         """
         return self.__default_weight
 
-    def get_default_size(self):
+    @staticmethod
+    def get_default_size():
         """
         Return the default font size.
         """
