diff -up matplotlib-1.2.0/lib/matplotlib/font_manager.py.orig matplotlib-1.2.0/lib/matplotlib/font_manager.py
--- matplotlib-1.2.0/lib/matplotlib/font_manager.py.orig	2013-02-24 22:50:54.932678576 -0300
+++ matplotlib-1.2.0/lib/matplotlib/font_manager.py	2013-02-24 22:54:29.627686797 -0300
@@ -62,7 +62,7 @@ try:
 except ImportError:
     import pickle
 
-USE_FONTCONFIG = False
+USE_FONTCONFIG = True
 
 verbose = matplotlib.verbose
 
@@ -771,7 +771,7 @@ class FontProperties(object):
                 return float(self._size)
             except ValueError:
                 pass
-        default_size = fontManager.get_default_size()
+        default_size = FontManager.get_default_size()
         return default_size * font_scalings.get(self._size)
 
     def get_file(self):
@@ -991,7 +991,10 @@ class FontManager:
         self.afmfiles = findSystemFonts(paths, fontext='afm') + \
             findSystemFonts(fontext='afm')
         self.afmlist = createFontList(self.afmfiles, fontext='afm')
-        self.defaultFont['afm'] = self.afmfiles[0]
+        try:
+            self.defaultFont['afm'] = self.afmfiles[0]
+        except IndexError:
+            self.defaultFont['afm'] = None
 
         self.ttf_lookup_cache = {}
         self.afm_lookup_cache = {}
@@ -1002,7 +1005,8 @@ class FontManager:
         """
         return self.__default_weight
 
-    def get_default_size(self):
+    @staticmethod
+    def get_default_size():
         """
         Return the default font size.
         """
@@ -1282,7 +1286,7 @@ if USE_FONTCONFIG and sys.platform != 'w
                     return file
         return None
 
-    _fc_match_regex = re.compile(r'\sfile:\s+"([^"]*)"')
+    _fc_match_regex = re.compile(br'\sfile:\s+"([^"]*)"')
     _fc_match_cache = {}
 
     def findfont(prop, fontext='ttf'):
diff -up matplotlib-1.2.0/lib/matplotlib/mpl-data/matplotlibrc.orig matplotlib-1.2.0/lib/matplotlib/mpl-data/matplotlibrc
--- matplotlib-1.2.0/lib/matplotlib/mpl-data/matplotlibrc.orig	2013-02-24 22:55:55.684690093 -0300
+++ matplotlib-1.2.0/lib/matplotlib/mpl-data/matplotlibrc	2013-02-24 22:57:00.146692561 -0300
@@ -198,7 +198,7 @@ text.hinting_factor : 8 # Specifies the
 #mathtext.it  : serif:italic
 #mathtext.bf  : serif:bold
 #mathtext.sf  : sans
-#mathtext.fontset : cm # Should be 'cm' (Computer Modern), 'stix',
+mathtext.fontset : stix # Should be 'cm' (Computer Modern), 'stix',
                        # 'stixsans' or 'custom'
 #mathtext.fallback_to_cm : True  # When True, use symbols from the Computer Modern
                                  # fonts when a symbol can not be found in one of
