--- setupext.py.orig	2014-01-25 15:06:39.460916454 +0100
+++ setupext.py	2014-01-25 15:06:53.080946205 +0100
@@ -841,15 +841,6 @@
     name = 'pycxx'
 
     def check(self):
-        if PY3:
-            # There is no version of PyCXX in the wild that will work
-            # with Python 3.x and matplotlib, since they lack support
-            # for the buffer object.
-            self.__class__.found_external = False
-            return ("Official versions of PyCXX are not compatible "
-                    "with matplotlib on Python 3.x, since they lack "
-                    "support for the buffer object.  Using local "
-                    "copy")
 
         self.__class__.found_external = True
         old_stdout = sys.stdout
