--- setupext.py.orig	2014-01-25 15:06:39.460916454 +0100
+++ setupext.py	2014-01-25 15:06:53.080946205 +0100
@@ -768,12 +768,6 @@
     name = 'pycxx'
 
     def check(self):
-        if sys.version_info[0] >= 3:
-            # There is no version of PyCXX in the wild that will work
-            # with Python 3.x
-            self.__class__.found_external = False
-            return ("Official versions of PyCXX are not compatible with "
-                    "Python 3.x.  Using local copy")
 
         self.__class__.found_external = True
         old_stdout = sys.stdout
