--- matplotlib-2.0.0rc2/lib/matplotlib/tests/test_axes.py	2016-12-18 11:40:53.000000000 -0800
+++ matplotlib-2.0.0rc2/lib/matplotlib/tests/test_axes.py.new	2017-01-07 21:35:06.874039829 -0800
@@ -361,7 +361,7 @@
     plt.plot('b','b', 'o', data=data)
 
 
-@image_comparison(baseline_images=['single_date'])
+@image_comparison(baseline_images=['single_date'], tol=1.97)
 def test_single_date():
     time1 = [721964.0]
     data1 = [-65.54]
@@ -4696,7 +4696,7 @@
 
 
 @image_comparison(baseline_images=['date_timezone_x_and_y'],
-                  extensions=['png'])
+                  extensions=['png'], tol=3.042)
 def test_date_timezone_x_and_y():
     # Tests issue 5575
     time_index = [pytz.timezone('UTC').localize(datetime.datetime(
--- matplotlib-2.0.0rc2/lib/matplotlib/tests/test_collections.py	2016-12-18 11:40:53.000000000 -0800
+++ matplotlib-2.0.0rc2/lib/matplotlib/tests/test_collections.py.new	2017-01-07 21:35:52.016235301 -0800
@@ -489,7 +489,7 @@
     ax.autoscale_view()
 
 
-@image_comparison(baseline_images=['polycollection_close'],
+@image_comparison(baseline_images=['polycollection_close'], tol=0.446,
                   extensions=['png'], remove_text=True)
 def test_polycollection_close():
     from mpl_toolkits.mplot3d import Axes3D
--- matplotlib-2.0.0rc2/lib/matplotlib/tests/test_contour.py	2016-12-18 11:40:53.000000000 -0800
+++ matplotlib-2.0.0rc2/lib/matplotlib/tests/test_contour.py.new	2017-01-07 21:36:29.283396673 -0800
@@ -233,7 +233,7 @@
             label.set_rotation(30)
 
 
-@image_comparison(baseline_images=['contour_test_label_transforms'],
+@image_comparison(baseline_images=['contour_test_label_transforms'], tol=0.731,
                   extensions=['png'], remove_text=True)
 def test_labels():
     # Adapted from pylab_examples example code: contour_demo.py
--- matplotlib-2.0.0rc2/lib/mpl_toolkits/tests/test_mplot3d.py	2016-12-18 11:40:53.000000000 -0800
+++ matplotlib-2.0.0rc2/lib/mpl_toolkits/tests/test_mplot3d.py.new	2017-01-07 21:37:04.144547626 -0800
@@ -171,7 +171,7 @@
     ax.set_zlabel('Z axis')
 
 
-@image_comparison(baseline_images=['trisurf3d'], remove_text=True, tol=0.03)
+@image_comparison(baseline_images=['trisurf3d'], remove_text=True, tol=0.081)
 def test_trisurf3d():
     n_angles = 36
     n_radii = 8
--- matplotlib-2.0.0rc2/lib/matplotlib/tests/test_transforms.py 2016-12-18 11:40:53.000000000 -0800
+++ matplotlib-2.0.0rc2/lib/matplotlib/tests/test_transforms.py.new     2017-01-07 21:21:29.478503151 -0800
@@ -82,7 +82,7 @@
 
 
 @image_comparison(baseline_images=['pre_transform_data'],
-                  tol=0.08)
+                  tol=0.15)
 def test_pre_transform_plotting():
     # a catch-all for as many as possible plot layouts which handle
     # pre-transforming the data NOTE: The axis range is important in this

