diff -up matplotlib-2.0.0b3/lib/matplotlib/sphinxext/tests/test_tinypages.py.tests matplotlib-2.0.0b3/lib/matplotlib/sphinxext/tests/test_tinypages.py
--- matplotlib-2.0.0b3/lib/matplotlib/sphinxext/tests/test_tinypages.py.tests	2016-07-17 01:04:11.000000000 +0200
+++ matplotlib-2.0.0b3/lib/matplotlib/sphinxext/tests/test_tinypages.py	2016-08-29 13:41:47.661198880 +0200
@@ -1,6 +1,7 @@
 """ Tests for tinypages build using sphinx extensions """
 
 import shutil
+import sys
 import tempfile
 
 from os.path import (join as pjoin, dirname, isdir)
@@ -16,6 +17,8 @@ TINY_PAGES = pjoin(HERE, 'tinypages')
 
 def setup():
     # Check we have the sphinx-build command
+    if sys.version_info[0] >= 3:
+        raise SkipTest('sphinx-build works only with python 2.x')
     try:
         ret = call(['sphinx-build', '--help'], stdout=PIPE, stderr=PIPE)
     except OSError:
